src/context/modal-context.js

```diff
@@ -6,20 +6,23 @@ const ModalContext = createContext()
 const ModalProvider = (props) => {
   const [modal, setModal] = useState()
   const [isFullScreen, setFullScreen] = useState(false)
+  const [paperProps, setPaperProps] = useState({})
```

Comment:
remove

Reply:
can't remove this, it`s needed to set props to context